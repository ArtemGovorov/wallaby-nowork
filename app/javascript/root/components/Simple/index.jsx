import React from 'react'
import styled from 'styled-components'

const Simple = styled.h1`
  color: orange;
  background: black;
`

export default Simple