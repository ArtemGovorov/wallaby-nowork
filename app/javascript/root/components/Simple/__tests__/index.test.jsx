import React from 'react'
import { shallow } from 'enzyme'
import Simple from 'root/components/Simple'

describe('<Simple />', () => {
  it('should render successfully', () => {
    const renderedComponent = shallow(<Simple />)
    expect(renderedComponent).toMatchSnapshot()
  })
})