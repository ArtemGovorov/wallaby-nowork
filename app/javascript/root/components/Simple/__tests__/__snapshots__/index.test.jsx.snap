// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Simple /> should render successfully 1`] = `
.c0 {
  color: orange;
  background: black;
}

<h1
  className="c0"
/>
`;
